### DISCUSSION TITLE
An Underworld3 docker images

### DISCUSSION THREAD

**Discussion by julesghub** [underworldcode/underworld3-archive#122] [Show and tell]
It's big and alpha, but it is here ... the first Underworld3 docker image! `julesg/underworld3:alpha`

Run it with something like (tweak the options as you see fit)
`docker run --rm -p 8999:8888 -v $HOME:/home/jovyan/workspace julesg/underworld3:alpha`

Get the running container name and get the access token with
`docker exec <containerName> jupyter server list`

Head to a browser, `localhost:8999` and you're good to go. A running JupyterLab with Underworld2 and Underworld3 installed side by side.

The UW team will be making changes to this container in future, so keep an eye on it. 
If you have any feedback/suggestions, we'd love to hear from you about it.

Happy modeling with Underworld3.

**Comment by jmansour** [comment]
Congrats guys!

**Reply by julesghub** [reply]
Thanks John!
